packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -69,57 +70,47 @@ class _TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
   @override
   void initState() {
     super.initState();
-    widget.data.expandCascading();
-    _flattenedList = _flattenExpandedTree();
-    columnWidths = _computeTableWidth();
-    _widthTween = Tween<double>(begin: tableWidth, end: tableWidth);
-    _resizeController = AnimationController(
-      vsync: this,
-      duration: const Duration(milliseconds: 200),
-    );
-    _resizeAnimation = CurvedAnimation(
-      parent: _resizeController,
-      curve: Curves.easeInOutCubic,
-    );
+    widget.data?.expandCascading();
+    _flattenedList = [];
+    _flattenExpandedTree(stack: _flattenedList);
+    columnWidths = _computeColumnWidths();
   }
 
   @override
   void didUpdateWidget(TreeTable<T> oldWidget) {
     super.didUpdateWidget(oldWidget);
-    _flattenedList = _flattenExpandedTree();
+    _flattenedList = [];
+    _flattenExpandedTree(stack: _flattenedList);
     refreshTree();
   }
 
-  @override
-  void dispose() {
-    _resizeController.dispose();
-    super.dispose();
-  }
-
-  List<T> _flattenExpandedTree({T node, bool Function(T node) filter}) {
+  void _flattenExpandedTree(
```

Comment:
nit: change method name from _flattenExpandedTree to a name that does not imply that the tree is being modified.
e.g. `_buildFlatTree` or similar

Reply:
Done.